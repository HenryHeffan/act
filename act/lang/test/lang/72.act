defproc test(chan(int) ichan; chan(bool) bchan)
{
  int a;
  bool b;
  chp {
     bchan?b;
     ichan?a;
     bchan?a;
     ichan?b
  }
}
