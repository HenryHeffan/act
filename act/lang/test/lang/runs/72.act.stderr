ERROR: File `72.act', line 8, col 12
	CHP receive: type-checking failed.
	Boolean/non-boolean types are incompatible.
